solcOptimiserSettings.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  enabled: true,
+  runs: 430,
```

Comment:
can set to 100000?

Reply:
Updated to 1M runs, will have warning for KyberNetwork contract in Mock folder